src/main/java/com/singh/astha/notificationservice/AppConfiguration.java

```diff
@@ -21,7 +21,7 @@
 @Configuration
 public class AppConfiguration {
 
-    private static final String[] SCOPES = {Constants.MESSAGING_SCOPE};
+    private static final String[] SCOPES = { Constants.MESSAGING_SCOPE };
```

Comment:
nit: The type can be changed to List<String> and the list can be initialised using List.of() and then the list can be passed directory to the GoogleCredentials.